addon/serializers/ilm-session.js

```diff
@@ -1,15 +1,8 @@
 import IliosSerializer from './ilios';
-import moment from 'moment';
 
 export default class IlmSessionSerializer extends IliosSerializer {
   serialize(snapshot, options) {
     const json = super.serialize(snapshot, options);
-    // set time to 5pm, always.
-    const dueDate = moment(json.data.attributes.dueDate);
-    dueDate.hour('17');
-    dueDate.minute('00');
-    json.data.attributes.dueDate = dueDate.format();
-
     //don't persist this, it is handled by the server
     delete json.data.attributes.updatedAt;
```

Comment:
`ilmSession` doesn't have an `updatedAt` property, I'm guessing this is an ancient copy/paste error from `session` and not this while file can go away.

Reply:
nice find. that allows us to remove the ilm-specific serializer altogether and use the default one instead  :heavy_check_mark: